sphericart-torch/python/tests/test_no_points.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import torch
+
+import sphericart.torch
+
+
+torch.manual_seed(0)
+
+
+@pytest.mark.parametrize("normalized", [False, True])
+@pytest.mark.parametrize("l_max", [0, 3, 7, 10, 20, 50])
+def test_error(l_max, normalized):
+    xyz = torch.zeros(0, 3, dtype=torch.float64)
+
+    calculator = sphericart.torch.SphericalHarmonics(l_max=l_max, normalized=normalized)
+
+    try:
+        sph = calculator.compute(xyz)
+    except RuntimeError as e:
+        pytest.fail(f"compute threw an error: {e}")
```

Comment:
Why are you doing this instead of just letting the exception go up the stack as usual? If you want to know which part failed, I would rather improve the original error messages

Reply:
the error messages that previously resulted from passing in a zero-sized tensor are that the pointers are not initialized. Since we force the zero-sized tensor here, I thought it was appropriate to report them this way, because this is the condition that we are testing for.